### DISCUSSION TITLE
manager后端服务启动时卡住无法成功启动

### DISCUSSION THREAD

**Discussion by notbugggg** [apache/hertzbeat#2921] [Q&A]
我fork仓库以后把代码pull下来，启动manager服务时控制台输出一直卡在Herzbeat图标、版本、port和pid不动，有人遇到过吗。我12.24时pull的代码还能启动，但是昨天再pull就启动不了了

**Comment by yunfan24** [comment]
Do you mean like this? I understand that this means the project has already started.
![image](https://github.com/user-attachments/assets/ca94fb48-5697-4b9f-b64a-44d2eaaf163b)
